maintainers/maintainer-list.nix

```diff
@@ -4165,6 +4165,12 @@
     githubId = 71959829;
     name = "Cleeyv";
   };
+  clemjvdm = {
```

Comment:
please split the maintainer list change into a separate commit, preceeding the commit that introduces the package